### ISSUE TITLE
Leave debugging on in the demonstration notebooks.

### ISSUE THREAD

**Issue by bangerth** [changliao1025/pyflowline_tutorial#4]
As discussed just now, when you ask people to change things and play with stuff, you really want debugging to be on to catch as many of the mistakes they make.

**Comment by changliao1025** [comment]
Yes, we will turn on the debug option for all online notebooks. Thank you!
